public/app/core/components/Login/LoginServiceButtons.tsx

```diff
@@ -46,6 +46,29 @@ export interface LoginServices {
   [key: string]: LoginService;
 }
 
+const LoginDivider = (keyNames: string[]) => {
+  const serviceElementsEnabled = keyNames.filter(key => {
+    const service: LoginService = loginServices()[key];
+    return service.enabled;
+  });
+  return serviceElementsEnabled.length > 0 ? (
```

Comment:
I would have moved this check into the main component function so that it does not render an empty login-oauth div (check if no services are enabled and return null at start of function)